### ISSUE TITLE
No output of CPAchecker in new view

### ISSUE THREAD

**Issue by leostrakosch** [SupunArunoda/CPACEP#31]
The new view of CPACEP doesn't show any output when running CPAchecker.
When pressing run, the bar at the top of the view changes from gray to yellow,
but there is never any output.

Instead, output should be displayed continuously.

Tested on Windows 10.

**Comment by SupunArunoda** [comment]
Hi Thomas,
Could you please let me know the output when the command execute on cpachecker by terminal?
Seems to be this has thrown unknown result.

**Comment by SupunArunoda** [comment]
Yes. This should be happend. When the result has Verification result TRUE the bar will be green color. If the result has FALSE, the bar will be red. If the result is unknown the bar will be yellow. This is the proposed appoach. Can you just try by keeping the default config and spec??

**Comment by leostrakosch** [comment]
My issue is that I don't see the output
```
CPAchecker 1.7-svn (Java HotSpot(TM) 64-Bit Server VM 1.8.0_171) started (CPAchecker.run, INFO)

More details about the verification run can be found in the directory ".\output".
```

in CPACEP, and there is no way to know whether analysis finished.
So do you get that output? And if not, do you know why it's not there?

**Comment by SupunArunoda** [comment]
Hi Thomas,
Currently CPACEP read the output by executing shell command using `Runtime.getRuntime().exec(command)`. But it cannot read the log information generated by CPAChecker. 
I have done the following configuration
![screenshot from 2018-07-03 23-39-36](https://user-images.githubusercontent.com/12041588/42237076-60603bc6-7f1a-11e8-9e54-2d6899821f5f.png)

Ant the CpacepView was as below.
![screenshot from 2018-07-03 23-40-31](https://user-images.githubusercontent.com/12041588/42237117-7ef40144-7f1a-11e8-8d4e-531cfa602d9f.png)

Given below the CPALog.txt generated by CPAchecker.
![screenshot from 2018-07-03 23-38-28](https://user-images.githubusercontent.com/12041588/42237166-aeacfb0c-7f1a-11e8-8230-ec2acd6471b4.png)

Seems like we need to append the log information to CpacepView too..

**Comment by leostrakosch** [comment]
Hi Supun,

I don't even see the output you're showing in the second screenshot. For me, the text view just stays disabled.

Regarding the log messages not showed: These are written to `stderr` instead of `stdout`,
that's why they are not displayed. But it should be possible to display these, too.
No need to read log files.

**Comment by SupunArunoda** [comment]
Hi Thomas,
Could you please share the related screenshots with me?? What will happen when set the spec and conf to other values??

**Comment by leostrakosch** [comment]
Hi Supun,

![image 1](https://user-images.githubusercontent.com/4499519/42292354-25eab5a8-7fd2-11e8-9411-05a6fdcc998c.jpg)
![image 2](https://user-images.githubusercontent.com/4499519/42292357-291449ce-7fd2-11e8-8189-287b9379ccda.jpg)

The temporary folder also stays empty, so maybe CPAchecker is never executed?
Unfortunately, there is no error message. Do you have any idea what the cause could be?
Do you have a windows machine available at the moment for debugging?

![image 3](https://user-images.githubusercontent.com/4499519/42292360-2c74878c-7fd2-11e8-9db8-ed7beb73b76b.jpg)


The same happens for the following configuration:

![image 4](https://user-images.githubusercontent.com/4499519/42292380-5a697d32-7fd2-11e8-9885-fa2ea5892a12.jpg)

**Comment by leostrakosch** [comment]
Fixed in above commit and should work if PR #29 is merged.
